frb_example/with_flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -376,7 +376,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.example.frbExample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				STRIP_STYLE = debugging;
+				STRIP_STYLE ="non-global";
```

Comment:
You can leave it alone, it has nothing to do with the test failure.
Just because https://github.com/fzyzcjy/flutter_rust_bridge/blob/5787939f15045990b691bdb31b137090362121cf/book/src/integrate/ios_headers.md